app/src/main/java/ba/sum/fpmoz/mim/ui/adapters/SubjectListAdapter.java

```diff
@@ -0,0 +1,69 @@
+package ba.sum.fpmoz.mim.ui.adapters;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.firebase.ui.database.FirebaseRecyclerAdapter;
+import com.firebase.ui.database.FirebaseRecyclerOptions;
+
+import ba.sum.fpmoz.mim.R;
+import ba.sum.fpmoz.mim.model.Subject;
+
+public class SubjectListAdapter extends FirebaseRecyclerAdapter<Subject, SubjectListAdapter.SubjectViewHolder1> {
+    public SubjectListAdapter(@NonNull FirebaseRecyclerOptions<Subject> options) { super(options); }
+
+    @Override
+    protected void onBindViewHolder(@NonNull SubjectListAdapter.SubjectViewHolder1 holder, int position, @NonNull Subject model) {
+        holder.subjectContent.setText(model.getName());
+
+    }
+
+    @NonNull
+    @Override
+    public SubjectViewHolder1 onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view=LayoutInflater.from(parent.getContext()).inflate(R.layout.viewholder_subject_to_class_admin,parent,false);
+        SubjectViewHolder1 viewHolder1=new SubjectListAdapter.SubjectViewHolder1(view);
+        viewHolder1.setOnClickListener(new Adapter.ClickListener() {
+            @Override
+            public void OnClickListener(View v, int position) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.